src/java/org/apache/cassandra/auth/AuthCache.java

```diff
@@ -212,6 +277,20 @@ public int getMaxEntries()
         return getMaxEntriesDelegate.getAsInt();
     }
 
+    public boolean getActiveUpdate()
+    {
+        return getActiveUpdate.getAsBoolean();
+    }
+
+    public void setActiveUpdate(boolean update)
+    {
+        if (Boolean.getBoolean("cassandra.disable_auth_caches_remote_configuration"))
+            throw new UnsupportedOperationException("Remote configuration of auth caches is disabled");
+
+        setActiveUpdate.accept(update);
+        cache = initCache(cache);
```

Comment:
nit: This is also happening above in `setMaxEntries()`, but the assignment and the call to `initCache()` together constitute a check-and-set operation. Throwing a `synchronized` on `setActiveUpdate()` and `setMaxEntries()` would at least make the static analysis tools happy.

Reply:
Looks like we also `initCache()` and assign in `setValidity()` and `setUpdateInterval()` as well. I see value in longer-blocking on these methods to make sure that, in the face of multiple operators calling JMX ops against these methods, we have a linearalizable set of operations that's occurred against the cache instead of indeterminate raciness.

Reply:
> multiple operators calling JMX ops against these methods

exactly